.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,29 @@
+name: Github Pages
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+          cache: "yarn"
+      - name: Install dependencies
+        run: yarn install
```

Comment:
```suggestion
        run: yarn
```

Reply:
Is it somehow possible to only install the subset of dependencies that are required for generating the docs?

Reply:
Not that I am aware of, though something like `npx` **could** help. Would only fetch the required program and execute it.

Reply:
Interesting, the dependencies are required because `TypedDoc` seems to be utilizing `tsc` under the hood, which itself requires all bindings to exist.

Reply:
OK makes sense. You could still remove the ` install`